### DISCUSSION TITLE
Where can I find a more recent correlation of trace log event breakdowns?

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#2687] [Documents and notes]
Sorry if I'm overlooking the location, but I can't figure out what combination of keywords to use to find it.

I find myself going to https://sandboxie-plus.github.io/sandboxie-docs/Content/SandboxieTrace.html a lot, but every time I get more confused since the trace log looks different enough that I can't follow along and eventually give up troubleshooting a process.

Could someone point me in the right direction where I can see a more recent break down? I could have sworn @isaak654 had something somewhere.

Things like:
```
|Process|      |Type|     |Status|      |Value|                     

02:36:50.603   File (D)   Closed (2)    \Device\HarddiskVolume11\   
02:36:50.603   File (U)   (4)        \Device\HarddiskVolume11\Users\User\krita-x64-5.1.5\share\color-schemes\KritaBlender.colors   
```
(D) is still a delete attempt?
(U) ?

**Comment by DavidXanatos** [comment]
(D) logged by Driver in kernel mode
(U) logged by sbiedll in User mode

**Reply by isaak654** [reply]
It's weird, because according to the documentation link in OP:
>  the character d logs requests which were denied. 

> a indicates if the access was allowed (A) or denied (D);

True or not, someone should update the documentation to clarify the ambiguity.

**Reply by DavidXanatos** [reply]
(A) and (D) reffer to the value column



|Process|                                 |Type|              |Status|   |Value|                                                                                                    

Start.exe (26924, 17956) - 02:01:17.342   File (D)            Trace      \Device\HarddiskVolume4\Projects\Sandboxie\SandboxiePlus\x64\Debug\SbieDll.dll (FA) 00120089.01.00000060   
Start.exe (26924, 17956) - 02:01:17.342   Ipc / Section (D)   Open       \KnownDlls\PSAPI.DLL (IA) 0000000D                                                                         
Start.exe (26924, 17956) - 02:01:17.344   Ipc (U)                        \Sessions\1\BaseNamedObjects\SBIE_BOXED_DummyEvent_26924                                                   
Start.exe (26924, 17956) - 02:01:17.344   Ipc / Event (D)                \Sandbox\DefaultBox\Session_1\Sessions\1\BaseNamedObjects\SBIE_BOXED_DummyEvent_26924 (IA) 001F0003        

there it is allowed  (A) or denied (D);

perhaps i should change the D in the type column dot (K)

**Reply by TFWol** [reply]
@DavidXanatos Thanks for the clarification.

@isaak654 Yeah, I've been trying to follow the docs, but it's a bit hard when I stare at a row in the Trace and wonder what I'm looking at. 
Once I have a good Key to reference something, I can get through quite a few things. Otherwise my brain goes in circles; I'm bad at guessing.

**Reply by isaak654** [reply]
> perhaps i should change the D in the type column dot (K)

As long as `File (K)` stands for Kernel mode, it's ok for me.
Having two D letters (even if in different columns) is confusing and the documentation is currently in an ambiguous state for this reason.

EDIT: this change has just been introduced in https://github.com/sandboxie-plus/Sandboxie/commit/8f2e3ac6abc699499136566dbd008de83dfc31de#diff-4e487faf4dd55249d5fd1aa4d0e71fc069a2bec75cd243dbae015ba2779f55e0
